content/docs/user-guide/editor/viewport/_index.md

```diff
@@ -1,40 +1,121 @@
+| Mode | Keyboard Hotkey | Description |
+| - | - | - |
+| World space | Hold **Shift** | Manipulators align with the unchanging world space that is common to all entities in a level. |
+| Parent space | | If an entity has a parent, manipulators align with the parent's rotation. |
+| Local space | | Manipulators align with the rotation of the entity that is selected. |
 
-**Topics**
+### Component mode switcher
 
-+ [Modifying the Transform](/docs/user-guide/editor/viewport/viewport-interaction-model-transform/)
-+ [Reference Space Model](/docs/user-guide/editor/viewport/viewport-interaction-model-reference-space-model/)
-+ [Using Group Selections and Reference Spaces](/docs/user-guide/editor/viewport/group-selections-reference-spaces/)
-+ [Resetting the Transform](/docs/user-guide/editor/viewport/reset-transform/)
-+ [Matching the Transform](/docs/user-guide/editor/viewport/transform-matching/)
-+ [Using Pivot Mode](/docs/user-guide/editor/viewport/pivot-mode/)
-+ [Using the Camera Space](/docs/user-guide/editor/camera-space/)
+![Image of component mode switcher widget](/images/user-guide/editor/component-mode-switcher.png)
+
+Every entity in the 3D viewport has a Transform component.  Other components that you can add to entities have properties that are easily edited in the 3D viewport using the component's _edit mode_.  You can **left-click** the icon of an editable component to enter that component's edit mode.  The Transform component's edit mode is selected by default. 
+
+The following table lists some of the components with an edit mode and describes the tasks you can complete while it is selected in the component mode switcher.  Refer to a component's page in the [Component Reference](/docs/user-guide/components/reference/) for a description of actions you can take while in edit mode.
+
+| Component | Edit Mode Tasks |
+| - | - |
+| [Shape](\docs\user-guide\components\reference\shape) | Edit the dimensions of the various shape components. |
+| [Whitebox](\docs\user-guide\components\reference\shape\white-box) | Design models with the Whitebox tool. |
+| [Spline](\docs\user-guide\components\reference\shape\spline) | Add, remove, and position spline control points. |
+| [Non-uniform Scale](\docs\user-guide\components\reference\non-uniform-scale) | Edit the scaling of an entity on the X, Y, and Z-axis separately. |
+| [PhysX Collider](\docs\user-guide\components\reference\physx\collider) | Edit the dimensions of a collider. |
+| [PhysX Ball Joint](\docs\user-guide\components\reference\physx\ball-joint) | Edit the position and limitations of a joint. |
+
+## Manipulators 
+
+With manipulators, you can edit an entity's Transform component properties directly in the 3D Viewport instead of in the [Entity Inspector](.\..\entity-inspector).  When you select an entity, only the current transform mode's manipulator displays.  The orientation of the manipulator matches the reference space selection.
```

Comment:
nit: Here we should either have `Transform Component` or `transform component` to be consistent (are we referring to the type or just generally talking about transforms and components).

Reply:
"...an entity's Transform component properties" - When we are writing about an actual ECS component, this is the correct style.  The component's name is capitalized but "component" is not.  [Relevant section of the style guide](https://www.o3de.org/docs/contributing/to-docs/style-guide/format/#bold-gems-and-components).

"...only the current transform mode's..." - The second occurrence of "transform" isn't referring to the ECS component so it isn't capitalized following that rule.  I think there is an argument to made that it should be capitalized as a proper noun, you can definitely find instances in the docs of "Game Mode" and "Editor Mode".  I don't think the style guide has defined what to do with terms like that yet.  On this page I've used an uncapitalized "transform mode" consistently though.